dbux-projects/src/userEvents/index.js

```diff
@@ -87,7 +89,7 @@ export function onUserEvent(cb) {
  * @param {Object} evtData NOTE: data *must* always be completely serializable, simple data.
  */
 export function emitUserEvent(eventType, evtData) {
-  if (manager.practiceSession) {
+  if (manager.practiceSession && !isStateFoundedType(manager.practiceSession.state)) {
```

Comment:
can we change `isStateFoundedType(manager.practiceSession.state)` to `practiceSession.isFinished()` etc?